pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -129,6 +129,10 @@ func (cc inboundChainConfig) ToFilterChainMatch(opt FilterChainMatchOptions) *li
 		} else {
 			match.PrefixRanges = IPv6PassthroughCIDR
 		}
+
+		if features.EnableDualStack && cc.clusterName == util.InboundPassthroughCluster {
+			match.PrefixRanges = IPv4PassthroughCIDR
```

Comment:
I am not sure why dual stack only support IPv4 prefix, how about pass through in with ipv6 dst address

Reply:
Yes, should add both IPv4 and IPv6. Will add.